microprofile/6.1/health/4.0.1/23.0.0.12-MicroProfile-Health-4.0.1-Java11-TCKResults.adoc

```diff
@@ -0,0 +1,177 @@
+:page-layout: certification 
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of MicroProfile Health 4.0.1
+
+== Open Liberty 23.0.0.12 - MicroProfile Health 4.0.1 Certification Summary (Java 11)
+
+* Product Name, Version and download URL (if applicable):
++
+https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/release/23.0.0.12/openliberty-23.0.0.12.zip[Open Liberty 23.0.0.12]
+
+* Specification Name, Version and download URL:
++
+https://github.com/eclipse/microprofile-health/tree/4.0.1[MicroProfile Health 4.0.1 Specification]
+
+* (Optional) TCK Version, digital fingerprint and download URL:
++
+https://repo1.maven.org/maven2/org/eclipse/microprofile/health/microprofile-health-tck/4.0.1/microprofile-health-tck-4.0.1.jar[MicroProfile Health 4.0.1 TCK]
++
+SHA-1: `0eb8df50272e27c93597ac98be94d60781556fc1`
++
+SHA-256: `3877b20e6a47223ef18609c427eeb4195d1ca78f19565ce6d8e71b5f80e3af8f`
+
+* Public URL of TCK Results Summary:
++
+xref:23.0.0.12-MicroProfile-Health-4.0.1-Java11-TCKResults.adoc[TCK results summary]
+
+
+* Java runtime used to run the implementation:
++
+Java 11: OpenJDK Runtime Environment (11.0.21+9)
+
+* Summary of the information for the certification environment, operating system, cloud, ...:
++
+Mac OS X: kern.version: Darwin Kernel Version 23.0.0: Fri Sep 15 14:42:42 PDT 2023; root:xnu-10002.1.13~1/RELEASE_X86_64
+
+Test results:
+
+[source, text]
+----
+Test suite: org.eclipse.microprofile.health.tck.JsonSchemaValidationTest 2023-12-02T03:20:00 UTC
+Tests:1 Failures:0 Errors:0
+   org.eclipse.microprofile.health.tck.JsonSchemaValidationTest.testPayloadJsonVerifiesWithTheSpecificationSchema_EE10_FEATURES_MicroProfile_61 Passed!
+Test suite: org.eclipse.microprofile.health.tck.HealthCheckResponseAttributesTest 2023-12-02T03:20:00 UTC
+Tests:1 Failures:0 Errors:0
+   org.eclipse.microprofile.health.tck.HealthCheckResponseAttributesTest.testSuccessResponsePayload_EE10_FEATURES_MicroProfile_61 Passed!
+Test suite: org.eclipse.microprofile.health.tck.SingleReadinessFailedTest 2023-12-02T03:20:00 UTC
+Tests:1 Failures:0 Errors:0
+   org.eclipse.microprofile.health.tck.SingleReadinessFailedTest.testFailureResponsePayload_EE10_FEATURES_MicroProfile_61 Passed!
+Test suite: org.eclipse.microprofile.health.tck.EnforceQualifierTest 2023-12-02T03:20:52 UTC
+Tests:1 Failures:0 Errors:0
+   org.eclipse.microprofile.health.tck.EnforceQualifierTest.testFailureResponsePayload_EE9_FEATURES_MicroProfile_50 Passed!
```

Comment:
Again EE9 MP5 results shouldn't be here I believe.  Same with 17.

Reply:
Will remove extra results.